docs/index.md

```diff
@@ -1,40 +1,84 @@
+    url: /compatibility
+    btn_label: Learn More
+    btn_class: "btn--primary"
+
+feature_row2:
+  - 
+    title: Syntax
+    image_path: /assets/images/example.png
+    excerpt: >
+      Luau is syntactically backwards-compatible with Lua 5.1 (code that is valid Lua 5.1 is also valid Luau); however, we have extended the language with a set of syntactical features that make the language more familiar and ergonomic. The syntax [is described here](syntax).
+    url: /syntax
+    btn:label: Learn More
+    btn_class: "btn--primary"
+
+feature_row3:
+  - 
+    title: Analysis
+    excerpt: >
+        To make it easier to write correct code, Luau comes with a set of analysis tools that can surface common mistakes. These consist of a linter and a type checker, colloquially known as script analysis, and can be used from [Roblox Studio](https://developer.roblox.com/en-us/articles/The-Script-Analysis-Tool) or using SECRET TOOL. The linting passes are [described here](lint), and the type checking user guide can [be found here](typecheck).
+    url: /typecheck
+    btn_label: Learn More
+    btn_class: "btn--primary"
+
+  - 
+    title: Performance
+    excerpt: >
+        In addition to a completely custom front end that implements parsing, linting and type checking, Luau runtime features new bytecode, interpreter and compiler that are heavily tuned for performance. Luau currently does not implement Just-In-Time compilation, but its interpreter is often competitive with LuaJIT interpreter on a wide set of benchmarks. We continue to optimize the runtime and rewrite portions of it to be even more efficient, including plans for a new garbage collector and further library optimizations, as well as an eventual JIT/AOT option. While our overall goal is to minimize the amount of time programmers spend tuning performance, some details about the performance characteristics are [provided for inquisitive minds](performance).
+    url: /performance
+    btn_label: Learn More
+    btn_class: "btn--primary"
+
+  -
+    title: Libraries
+    excerpt: >
+        As a language, Luau is a full superset of Lua 5.1. As far as standard library is concerned, some functions had to be removed from the builtin libraries, and some functions had to be added. Additionally, Luau is currently only runnable from the context of the Roblox engine, which exposes a large API surface [documented on Roblox developer portal](https://developer.roblox.com/en-us/api-reference).
+
+---
+
+{% include feature_row id="feature_row1" %}
+
+{% include feature_row id="feature_row2" type="left" %}
+
+{% include feature_row id="feature_row3" %}
+
+### Terms of Use
+
+See [here](limited-terms-of-use) for all of the important legal stuff.
```

Comment:
fwiw I believe this one relates exclusively to Studio betas, which I think we will take down once we launch the public beta of parallel execution, so we don't have to link to this; this currently contains language that's somewhat specific to the beta program including use of Roblox account et al.